project/plugins.sbt

```diff
@@ -10,5 +10,6 @@ resolvers += Resolver.url(
   "Broad Artifactory",
   new URL("https://broadinstitute.jfrog.io/broadinstitute/libs-release/")
 )(publishPatterns)
+resolvers += Resolver.sonatypeRepo("releases")
```

Comment:
i think we can remove this since i didn't end up using the library that required this